File: app/src/main/java/com/rinconadadelsur/sistemas/bioseguridad/Entidades/eGalpones.java
package com.rinconadadelsur.sistemas.bioseguridad.Entidades;

public class eGalpones {
    private String _tcencos;
    private String _tcodint;

    public eGalpones(){
    }

    public eGalpones(String _tcencos, String _tcodint) {
        this._tcencos = _tcencos;
        this._tcodint = _tcodint;
    }

    public String get_tcencos() {
        return _tcencos;
    }

    public void set_tcencos(String _tcencos) {
        this._tcencos = _tcencos;
    }

    public String get_tcodint() {
        return _tcodint;
    }

    public void set_tcodint(String _tcodint) {
        this._tcodint = _tcodint;
    }

}

File: app/src/main/java/com/rinconadadelsur/sistemas/bioseguridad/Entidades/eGalpones.kt
package com.rinconadadelsur.sistemas.bioseguridad.Entidades

class eGalpones {
    private var _tcencos: String? = null
    private var _tcodint: String? = null

    constructor()

    constructor(_tcencos: String?, _tcodint: String?) {
        this._tcencos = _tcencos
        this._tcodint = _tcodint
    }

    fun get_tcencos(): String? {
        return _tcencos
    }

    fun set_tcencos(_tcencos: String?) {
        this._tcencos = _tcencos
    }

    fun get_tcodint(): String? {
        return _tcodint
    }

    fun set_tcodint(_tcodint: String?) {
        this._tcodint = _tcodint
    }
}
